tests/upgrade_path/upgrade_helpers.py

```diff
@@ -59,6 +60,32 @@ def check_sonic_version(duthost, target_version):
         "Upgrade sonic failed: target={} current={}".format(target_version, current_version)
 
 
+def restore_image_to_first_boot(duthost, image_version):
```

Comment:
@saiarcot895 define the function helper what it does and what it return value mean?

Reply:
Added description.